File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/data/api/UserResponse.java
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api;


public class UserResponse {

    public String username;
    public String objectId;
    public String email;
    public String firstName;
    public String lastName;
    public String sessionToken;

    @Override
    public String toString() {
        return "UserResponse{" +
                "username='" + username + '\'' +
                ", objectid='" + objectId + '\'' +
                ", email='" + email + '\'' +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", sessionToken='" + sessionToken + '\'' +
                '}';
    }
}

File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/data/api/response/UserResponse.kt
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.data.api.response

data class UserResponse(
    val username: String,
    val objectId: String,
    val email: String,
    val firstName: String,
    val lastName: String,
    val sessionToken: String
)
